packages/trader/src/AppV2/Components/SymbolNotFound/symbol-not-found.tsx

```diff
@@ -0,0 +1,25 @@
+import { Text } from '@deriv-com/quill-ui';
+import { StandaloneSearchRegularIcon } from '@deriv/quill-icons';
+import { Localize } from '@deriv/translations';
+import React from 'react';
+
+const SymbolNotFound = ({ searchTerm }: { searchTerm?: string }) => {
+    return (
+        <div className='symbol-not-found--container'>
```

Comment:
This isn't a modifier. If you consider it a block, the className should be `symbol-not-found-container`. If you consider it an element, it should be `symbol-not-found__container`, and then the children should follow this convention.